core/camel-core-model/src/main/java/org/apache/camel/model/SendDefinition.java

```diff
@@ -106,7 +109,18 @@ public String getPattern() {
 
     @Override
     public String getLabel() {
-        String uri = getEndpointUri();
+        if (endpointUriToString == null) {
+            String value = null;
+            try {
+                value = getEndpointUri();
+            } catch (RuntimeException e) {
+                // ignore any exception and use null for building the string value
```

Comment:
Why do we need to catch the exception now knowing it was not needed before?

Reply:
Thats how the code is done elsewhere, its just for a lavel

Reply:
See DefaultEndpoint I think it is